tests/headingbuttonsui.js

```diff
@@ -24,7 +24,18 @@ describe( 'HeadingButtonUI', () => {
 			return ClassicTestEditor
 				.create( editorElement, {
 					plugins: [ HeadingButtonsUI, HeadingEditing ],
-					toolbar: [ 'heading1', 'heading2', 'heading3' ]
+					toolbar: [ 'heading1', 'heading2', 'heading3', 'heading3' ],
```

Comment:
why use heading3 two times?

Reply:
Totally a typo :) Just to be sure - this change is either not needed (the buttons are not tested via this config AFAICS or other buttons should be added.

Reply:
Yeah, I somehow missed that. 

To stay on the safe side I added all buttons to the toolbar configuration in 5175878. FYI it's not necessary because the [following test](https://github.com/ckeditor/ckeditor5-heading/blob/t/121/tests/headingbuttonsui.js#L54:L63) checks all the buttons anyway but I figured we should stay consistent.